recipes/pybufr-ecmwf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in commenets in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.81" %}
+{% set sha256 = "abe0f47bfcac9603c4b47a9dc7d95befed6ae3091ef87f2511e42bbfa5b52e35" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#   $ openssl sha256 <file>
+# On Windows, you'll need the openssl pacakge, available on conda-forge
+
+package:
+  name: pybufr-ecmwf
+  version: {{ version }}
+
+source:
+  fn: pybufr-ecmwf-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/pybufr-ecmwf/pybufr-ecmwf-{{ version }}.tar.gz
+  # md5 is OK but sha256 is desirable.
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win]
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - numpy x.x
+    - python
+    - libgfortran  # [osx]
```

Comment:
You need the `gcc` package here to get a `fortran` compiler. And you you need `libgfortran` on `OS X` and maybe `libgcc` on Linux (try `libgfortran` on Linux too, but if that fails due to a missing `libquadmath` add `libgcc`.)